packages/react-native/src/primitives/TextField/styles.ts

```diff
@@ -19,23 +19,22 @@ export const getThemedStyles = (theme: StrictTheme): TextFieldStyles => {
       opacity: opacities[60],
       ...components?.textField?.disabled,
     },
-    inputContainer: {
+    fieldContainer: {
       alignItems: 'center',
       borderColor: colors.border.primary,
       borderRadius: radii.small,
       borderWidth: borderWidths.small,
       flexDirection: 'row',
       lineHeight: getLineHeight(fontSizes.medium),
       padding: space.medium,
-      width: '100%',
```

Comment:
Will this impact existing `TextField` usage?

Reply:
I tested our authenticator screens and didn't see impact.